sxiv.h

```diff
@@ -414,6 +424,10 @@ struct win {
 	XftColor bar_bg;
 	XftColor bar_fg;
 
+	suffixmode_t title_suffixmode;
+	const char   *title_prefix;
+	const char   *title_suffix;
+
```

Comment:
Do we need this? We already have `title_prefix` and `title_suffixmode` inside `opt`.